one-click-integration.sh

```diff
@@ -176,7 +176,7 @@ echo ""
 ## Decode response and save as a certificate file
 echo "Decoding and saving certificate"
 echo ""
-crt=$( echo ${crtb64} | base64 --decode > generated.crt )
+crt=$(echo "${crtb64}" | base64 -d > generated.crt )
```

Comment:
I've got an error running this on Mac OS, the -d option does not work. After the following change I was able to go through the whole flow:
```suggestion
crt=$(echo "${crtb64}" | base64 -D > generated.crt )
```

Is it also a viable solution for Linux?

Reply:
It does not, however we could use openssl for that: `openssl enc -base64 -d`